### PR TITLE
attrib target blank added to version link

### PR DESCRIPTION
**Pull request by vinay-ebi**

### CODE REVIEW

File: src/ensembl/production/handover/templates/base.html [hunk 1]
```diff
@@ -38,8 +38,8 @@
         {% block content %}
             <nav class="navbar navbar-icon-top navbar-expand-lg navbar-dark hnav ">
                 <a class="navbar-brand" href="#"><span class="mb-0 h3">Handover {{config['HANDOVER_TYPE']}} </span></a>
-                    <ul href="https://github.com/Ensembl/ensembl-prodinf-handover/blob/release/{{config['APP_VERSION']}}/CHANGELOG.md" class="nav navbar-nav navbar-right navbar-toggler" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation" >
-                        <li><a class="nav-item nav-link active">e{{config['RELEASE']}} - v{{config['APP_VERSION']}}</a></li>
+                    <ul  class="nav navbar-nav navbar-right navbar-toggler" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation" >
```
Comment (marcoooo) [inline src/ensembl/production/handover/templates/base.html:41]: The href is on a <ul> tag, is it expected?

**Review by marcoooo — CHANGES_REQUESTED** [review]

File: src/ensembl/production/handover/templates/base.html (hunk 1 above)
Comment (vinay-ebi) [reply src/ensembl/production/handover/templates/base.html:41]: its only displayed on mobile view

Comment (marcoooo) [reply src/ensembl/production/handover/templates/base.html:41]: I seriously doubt anyone would use the mobile view! You may retire it.

Comment (vinay-ebi) [reply src/ensembl/production/handover/templates/base.html:41]: sure

**Review by marcoooo — APPROVED** [review]

**Pull request merged**